### PR TITLE
feat: Block Sanctioned Addresses

### PR DESCRIPTION
**Pull request by jwhardwick**
### Hi👋, please prefix this PR's title with:

- [ ] `breaking-change:` if you have introduced modification that necessitates immediate adjustments by this SDK's users to their applications, clients, or integrations to avert disruptions to existing features or functionalities.
- [x] `feat:`, `fix:`, `refactor:`, `docs:`, or `chore:`.

# Summary

Blocks users that are sanctioned from accessing Checkout Widgets.
https://immutable.atlassian.net/browse/CM-793

image

### CODE REVIEW

File: packages/checkout/widgets-lib/src/views/error/serviceTypes.ts [hunk 1]
```diff
@@ -1,3 +1,4 @@
 export enum ServiceType {
   SWAP = 'swap',
+  SANCTION = 'sanction',
```
Comment (jwhardwick) [inline packages/checkout/widgets-lib/src/views/error/serviceTypes.ts:3]: Not particularly happy with naming here. Want to re-use ServiceUnavailable view though

File: packages/checkout/widgets-lib/src/locales/en.json [hunk 2]
```diff
@@ -70,10 +70,13 @@
     },
     "SERVICE_UNAVAILABLE_ERROR_VIEW": {
       "heading": {
-        "swap": "Swapping is not available in your region"
+        "swap": "Swapping is not available in your region",
+        "sanction": "Wallet is sanction placeholder copy"
```
Comment (jwhardwick) [inline packages/checkout/widgets-lib/src/locales/en.json:74]: Needs proper copy

File: packages/checkout/widgets-lib/src/locales/en.json [hunk 3]
```diff
@@ -70,10 +70,13 @@
...
       },
       "body": {
-        "swap": "Please refer to <quickswapLink>Quickswap’s</quickswapLink> website for further information."
+        "swap": "Please refer to <quickswapLink>Quickswap’s</quickswapLink> website for further information.",
+        "sanction": "Please refer to PLACEHOLDER COPY for more information"
```
Comment (jwhardwick) [inline packages/checkout/widgets-lib/src/locales/en.json:78]: Needs translated copy

**Review by mimi-imtbl — APPROVED** [review]

**Review by jhesgodi — APPROVED** [review]

**Comment by jwhardwick** [comment]
Validated these changes on Sandbox, working with MM and Passport.

**Pull request merged**